src/createBrowserLikeFetch.js

```diff
@@ -70,6 +71,13 @@ function createBrowserLikeFetch({
               return;
             }
 
+            if (!cookieOptions.domain) {
```

Comment:
So if we exclude the domain from the server and the request is made from localhost, will this allow the cookie to be set?

Reply:
if the request is made to `http://localhost` and the server's response doesn't include the Domain attribute in the Set-Cookie header...yes, good point, I didn't think about that 😅
IIRC localhost is treated a bit different, but at the moment I forget how
is this desired behavior? I guess it would be, localhost is under the control of the user so if they're using it for development running various servers they'd want those cookies?

Reply:
Actually this is desired and IIRC it is the way to test cookies when developing locally. I remember I had to disable the domain suffix validation before so I could get passed this issue where my cookies were not set in localhost

Reply:
I should have run it
```
> require('tough-cookie').getPublicSuffix('localhost')
null
> require('tough-cookie').getPublicSuffix(new URL('http://localhost/the/thing/you/know').hostname)
null
```
perhaps we need a special case for `localhost`? only when `NODE_ENV=development`? seems like a new PR? 😅

Reply:
Yeah it would be nice as a separate feature